payjoin-cli/src/main.rs

```diff
@@ -18,22 +24,14 @@ async fn main() -> Result<()> {
             let bip21 = sub_matches.get_one::<String>("BIP21").context("Missing BIP21 argument")?;
             let fee_rate_sat_per_vb =
                 sub_matches.get_one::<f32>("fee_rate").context("Missing --fee-rate argument")?;
-            #[cfg(feature = "v2")]
-            let is_retry = matches.get_one::<bool>("retry").context("Could not read --retry")?;
-            #[cfg(feature = "v2")]
+            let is_retry = matches.get_one::<bool>("retry").unwrap_or(&false);
             app.send_payjoin(bip21, fee_rate_sat_per_vb, *is_retry).await?;
-            #[cfg(not(feature = "v2"))]
-            app.send_payjoin(bip21, fee_rate_sat_per_vb).await?;
         }
         Some(("receive", sub_matches)) => {
             let amount =
                 sub_matches.get_one::<String>("AMOUNT").context("Missing AMOUNT argument")?;
-            #[cfg(feature = "v2")]
-            let is_retry = matches.get_one::<bool>("retry").context("Could not read --retry")?;
-            #[cfg(feature = "v2")]
+            let is_retry = matches.get_one::<bool>("retry").unwrap_or(&false);
```

Comment:
are you planning to take this out in a different PR?

Reply:
Yes, I'll follow up with it's removal. This was focused on the module separation only